charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -444,6 +453,9 @@ spec:
             - --kube-api-qps=20
             - --kube-api-burst=100
             - --workers=100
+            {{- end }}
+             {{- if not (regexMatch "(-retry-interval-start)" (join " " .Values.sidecars.resizer.additionalArgs)) }}
```

Comment:
nitpick: remove one extra space in the indent here

Reply:
Ahhh man was practicing my vim skills and must have missed it thank you will fix in rev 2